components/nexusoperations/frontend/handler.go

```diff
@@ -524,3 +525,11 @@ func (c *requestContext) shouldForwardRequest(ctx context.Context, header http.H
 		len(c.namespace.ClusterNames()) > 1 &&
 		c.Config.ForwardingEnabledForNamespace(c.namespace.Name().String())
 }
+
+// enrichNexusOperationMetrics enhances metrics with additional Nexus operation context.
+func (c *requestContext) enrichNexusOperationMetrics(service, operation string) {
+	c.metricsHandler = c.metricsHandler.WithTags(
+		metrics.NexusServiceTag(service),
+		metrics.NexusOperationTag(operation),
+	)
```

Comment:
The other `enrich` function checks dynamic configs to add the tags. I think you should do the same.

Reply:
added dynamic config via dependency injection